libraries/AP_HAL_ChibiOS/hwdef/3DRControlN1/hwdef-bl.dat

```diff
@@ -0,0 +1,50 @@
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# USB setup
+USB_VENDOR 0x26ac
```

Comment:
same comment...should use defaual VID/PID unless you add drivers to all GC install packages

Reply:
We have added these in the past for CZ OEM